target/linux/mvebu/image/Makefile

```diff
@@ -212,4 +225,17 @@ define Device/globalscale-mirabox
 endef
 TARGET_DEVICES += globalscale-mirabox
 
+define Device/turris-omnia
+  $(Device/NAND-256K)
+  KERNEL_INSTALL := 1
+  KERNEL := dtb | kernel-bin
+  DEVICE_DTS := armada-385-turris-omnia
+  DEVICE_TITLE := Turris Omnia
+  DEVICE_PACKAGES := swconfig wpad-mini kmod-ath9k kmod-ath10k ath10k-firmware-qca988x
```

Comment:
Case for exclusion: It increases the size of images and take some space for users that don't have WiFi modules.

Case for inclusion: The Omnia has a very large amount of storage and I assume that a majority shipped with the WiFi modules. <1MB out of 8GB is negligible.

I think it is more likely that space-conscious users will be building their own images, while those downloading images with the default configuration will be expecting some level of functionality out of the box.

There, I have expressed my opinion. Hopefully it's settled faster than the last time I was involved in this discussion. #931